1.8/administration/access-node/tunnel.md

```diff
@@ -182,24 +209,23 @@ Alternatively, you can add `name` to the `portMappings` or `portDefinitions` fie
   ]
 ```
 
-## VPN
-Run the following command from the DC/OS CLI
+##  Using DC/OS Tunnel to run a VPN
+1. Run the following command from the DC/OS CLI
```

Comment:
Sorry, I don't think the `1.` is needed on this last step - since there's only a single step.